### ISSUE TITLE
Web support

### ISSUE THREAD

**Issue by watadarkstar** [heylinda/heylinda-app#10]
We can use Expo for this, see:
https://github.com/heylinda/heylinda-app/blob/main/package.json#L7

We probably just need to host it somewhere.

**Comment by thatretrodev** [comment]
To be clear,
- I just saw this not long before I wrote this comment
- I pointed out this problem on discord
- I gave the error report on discord
- I did not request to be anonymous or something, in fact I was going to create this issue myself
- And most importantly, I seriously want web support!

**Comment by YajanaRao** [comment]
Running on web gives the following error:
![image](https://user-images.githubusercontent.com/38914337/127537413-e7adee19-4659-403e-95d6-f9f8f29a8327.png)

**Comment by watadarkstar** [comment]
@YajanaRao Yup I think its the same or similar issue as pointed out by @thatretrodev :)

**Comment by YajanaRao** [comment]
The issue related to react native calendars web support is closed in the official repo: 
https://github.com/wix/react-native-calendars/issues/2

**Comment by YajanaRao** [comment]
I found this library https://www.npmjs.com/package/react-native-paper-dates which has support for web as well. but not sure weather it can replace the existing react-native-calenders library

**Comment by thatretrodev** [comment]
Hello! I pointed out this problem a while ago not thinking that people would come here and try really hard to fix it (This Repository also has gotten ***a lot*** more stars since then), but somehow that did happen.
Anyways, Since I'm already making this comment........


> Running on web gives the following error:
> ![image](https://user-images.githubusercontent.com/38914337/127537413-e7adee19-4659-403e-95d6-f9f8f29a8327.png)

That is the same error.


> I found this library https://www.npmjs.com/package/react-native-paper-dates which has support for web as well. but not sure weather it can replace the existing react-native-calenders library

I like the UI in that library.

**Comment by YajanaRao** [comment]
As react native paper is already used in the app react native paper dates matches with the theme of the application. I doubt that if react native paper dates can replace react native calendars.

**Comment by YajanaRao** [comment]
Able to get the application by downgrading react-native-calenders version to `1.1260.0`

![image](https://user-images.githubusercontent.com/38914337/128625803-865ab241-722a-41d0-add2-35f850d85264.png)

**Comment by thatretrodev** [comment]
> Able to get the application by downgrading react-native-calenders version to `1.1260.0` based on the solution given here [wix/react-native-calendars#1518](https://github.com/wix/react-native-calendars/issues/1518)
> 
> ![image](https://user-images.githubusercontent.com/38914337/128625803-865ab241-722a-41d0-add2-35f850d85264.png)

Quite funny that you posted that right before https://github.com/wix/react-native-calendars/pull/1548 was merged

**Comment by YajanaRao** [comment]
Shall I just create a PR with the changes mentioned above @watadarkstar ?

**Comment by watadarkstar** [comment]
@YajanaRao Since this got merged wix/react-native-calendars#1548 can we upgrade the library and it should work?

**Comment by YajanaRao** [comment]
Getting this error now on web:
![image](https://user-images.githubusercontent.com/38914337/136060413-fa7b9390-a58e-4d24-aafc-ed0cb21309b4.png)

With below version of the package
```
"react-native-calendars": "^1.17.0",
```

**Comment by YajanaRao** [comment]
What do you mean by disable offline support for web ? I am thinking of doing a conditional import or to something like below which can fix the issue for now:
```js
const files = Platform.OS != "web" ? useFiles('.mp3') : []
```

**Comment by watadarkstar** [comment]
@YajanaRao Yes like that but instead of doing it inside the component do that logic inside the hook. Like so:

```
export const useFiles = async (filetype: string) => {
  if (!filetype || Platform.OS === 'web') {
    return undefined
  }
...
}
```
